homeassistant/components/denonavr/media_player.py

```diff
@@ -247,9 +247,29 @@ def __init__(
             and MediaPlayerEntityFeature.SELECT_SOUND_MODE
         )
 
+        self._receiver.register_callback("ALL", self._telnet_callback)
+
+    async def _telnet_callback(self, zone, event, parameter):
```

Comment:
In the future it would be nicer if the library accepted a normal function to be run as the callback instead of a coroutine as its much more efficient and nothing is being awaited here.

Reply:
I originally had it this way, actually I made it accept a normal function or a coroutine but the owner of the library wanted only coroutines.